homeassistant/components/ialarmxr/translations/it.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Please remove this translation, you can generate the English translation for testing purposes, but all other translations will be done via Lokalise platform.

Reply:
Uh ok 👍 ...I'm italian native so  it's a "trusted" translation 😅 but it's ok, I'll remove it 👍